JASMIN/catalog-JASMIN.yaml

```diff
@@ -42,3 +42,46 @@ sources:
         max: 2015
         min: 1958
         type: int
+
+  ORCHESTRA_ZPS_CORE2NYF:
+    args:
+      urlpath: /bodc/BAS210039/CORE2NYF-ORCH0083-LIM3/{{ year }}/{{ time_freq }}/{{ grid }}/*.nc
+      xarray_kwargs:
+        compat: override
+        coords: minimal
+        data_vars: minimal
+        decode_timedelta: false
+        parallel: parallel
```

Comment:
@atb299 Do this open_mfdataset settings look good? Do you think we should add or change something?